boggs/src/main/java/com/hartwig/healthchecks/boggs/adapter/BoggsAdapter.java

```diff
@@ -18,6 +13,10 @@
 import com.hartwig.healthchecks.common.resource.ResourceWrapper;
 import com.hartwig.healthchecks.common.util.BaseReport;
 import com.hartwig.healthchecks.common.util.CheckCategory;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.IOException;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Wrong order for 'java.io.IOException' import.](https://www.codacy.com/app/ekho/health-checks/file/2773600378/issues/source?bid=3363458&fileBranchId=3363544#l19)